### PR TITLE
Fix typo in script.js.

### PR DESCRIPTION
**Pull request by Sami-2000**
Add a space where necessary.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ function bubbleFall() {
     const id = setInterval(lowerBubbleOnePixel, 10);
     // For each movement, move the element one pixel down. At the bottom, stop moving.
     function lowerBubbleOnePixel() {
-        if (height == 1000){
+        if (height == 1000) {
             clearInterval(id);
         }
         else{
```
Comment (billyeh) [inline portfolio/src/main/webapp/script.js]: thanks! Here too.

**Review by billyeh — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (Sami-2000) [reply portfolio/src/main/webapp/script.js]: I don't know why, but on my version that change is already completed. I will try pushing again.

Comment (Sami-2000) [reply portfolio/src/main/webapp/script.js]: It actually won't let me commit again because it says there have been no changes.

Comment (billyeh) [reply portfolio/src/main/webapp/script.js]: SG! Yeah maybe some unpushed commit or uncommited chnage?

Comment (Sami-2000) [reply portfolio/src/main/webapp/script.js]: It will not let me commit or push again :(

**Pull request merged**